urdf_filename: "/home/vsraghavan/bigman.urdf"
joint_names: ["LWrj2","LWrj1", "LForearmPlate", "LElbj", "LShYaw", "LShLat", "LShSag", "WaistYaw", "WaistSag", "WaistLat" ]
waist_in_optitrack: "Waist"
file_save_name: "/home/vsraghavan/bigman_test.urdf"
hand_in_optitrack: "OptiHand"
